http_server/_server_conn_handler.pony

```diff
@@ -1,5 +1,5 @@
-use "net"
 use "debug"
+use "net"
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled back